server/management/documents/appointmentsFilingStatuses.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+$root = realpath($_SERVER["DOCUMENT_ROOT"]);
+require_once "$root/server/user.class.php";
+$USER = new User();
+if (!$USER->hasPermission('use_admin_tools')) {
+	header("Location: /unauthorized");
+	die();
+}
+$HEADER_COLUMN_NAMES = array('First Name', 'Last Name', 'Appointment ID', 'Appointment Completed', 'Reason if Not Completed', 'State E-File', 'Federal E-File', 'State Paper', 'Federal Paper');
+$ALL_SITES_ID = -1;
+
+require_once "$root/server/config.php";
+require_once "$root/server/libs/wrappers/PHPExcelWrapper.class.php";
+
+getAppointmentsFilingStatusesExcelFile($_GET);
+
+function getAppointmentsFilingStatusesExcelFile($data) {
+	$appointments = executeAppointmentQuery($data);
+	$phpExcelWrapper = createAppointmentsFilingStatusesExcelFile($appointments);
+
+	ob_clean();
+	ob_end_clean();
+
+	$fileName = $data['date'] . '_AppointmentFilingStatuses' . '.xlsx';
+	header('Content-Type: application/vnd.openxmlformats-officedocument.spreadsheetml.sheet');
+	header('Content-Disposition: attachment;filename="'. $fileName .'"');
+	header('Cache-Control: max-age=0');
+	header ('Expires: Mon, 26 Jul 1997 05:00:00 GMT');
+	header ('Last-Modified: '.gmdate('D, d M Y H:i:s').' GMT');
+	header ('Cache-Control: cache, must-revalidate');
+	header ('Pragma: public');
+
+	$objWriter = $phpExcelWrapper->createExcelWriter();
+	$objWriter->save('php://output');
+
+	exit;
+}
+
+function executeAppointmentQuery($data) {
+	GLOBAL $DB_CONN, $ALL_SITES_ID;
+
+	$query = 'SELECT Client.firstName, Client.lastName, Appointment.appointmentId,
+		ServicedAppointment.completed, ServicedAppointment.notCompletedDescription, ServicedAppointment.servicedAppointmentId,
+		Site.title
```

Comment:
Change to ```Site.title, Site.siteId```

Reply:
Whoops, good catch, got a little too delete-happy when I was writing this query originally